floris/tools/optimization/scipy/yaw.py

```diff
@@ -388,14 +335,15 @@ def maximum_yaw_angle(self, value):
     @property
     def x0(self):
         """
-        This property gets or sets the initial yaw angles for the 
-        optimization.
+        Gets or sets the initial yaw angles used for the optimization.
+
+        **Note**: This is a virtual property used to "get" or "set" a value.
         
         Args:
-            value (float): The initial yaw angles (deg).
+            value (iterable): The yaw angle initial conditions to set (deg).
```

Comment:
`(iterable)` ... good point. This is the more general form rather than specifying `np.array` or `list`.